src/Tribe/Admin/Notices.php

```diff
@@ -329,4 +331,60 @@ function maybe_display_paystack_notice() {
 		);
 
 	}
+
+	/**
+	 * Display AR Page notice for FSE theme detected.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_display_fse_ar_page_notice() {
+		// Bail if we aren't in Tickets > Settings.
+		if ( Settings::$settings_page_id !== tec_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if Tickets Plus is not active.
+		if ( ! did_action( 'tec_container_registered_provider_Tribe__Tickets_Plus__Service_Provider' ) ) {
+			return;
+		}
+
+		// Bail if not using block theme.
+		if ( ! wp_is_block_theme() ) {
+			return;
+		}
+
+		// Bail if the attendee registration page is already set or does not exist.
+		$id = Tribe__Settings_Manager::get_option( 'ticket-attendee-page-id', false );
+		if ( ! empty( $id ) || ! get_post_status( $id ) ) {
```

Comment:
I would specifically check against `publish` So that would need to become:

```suggestion
		if ( ! empty( $id ) && 'publish' === get_post_status( $id ) ) {
```

So we bail if id exists and the status is publish only.

Reply:
Meh. We just want to make sure it exists. I don't feel like we need to go as far as checking the status, checking if it has a shortcode, punctuation, spelling, etc.

Reply:
Alright so it would still need to become like below otherwise you are bailing when the get_post_status returns false.

```suggestion
		if ( ! empty( $id ) && get_post_status( $id ) ) {
```

Reply:
If the `$id` is empty, we want to bail... OR if the `get_post_status() === false` we want to bail. I think if we used `&&`, it would throw a fatal if `$id` was `null`.

Reply:
It wouldn't throw a fatal because then it would be empty as a result it would not be evaluating the get_post_status at all.

Why do we need to bail if get_post_status is false ? It means that the $id is present but no post status, so the post has been deleted. As a result the $id is equivalent to empty.

Following the above logic it would return 'trash' or 'draft' or 'private' as well so i think we should follow my initial comment [here](https://github.com/the-events-calendar/event-tickets/pull/3342#discussion_r1821559598)